@codex please fix

```
> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
 ✓ tests/eslintPluginPresence.test.ts (1 test) 6ms
 ✓ frontend/tests/gameStateFlush.test.ts (1 test) 5ms
 ✓ tests/checkoutFetchDepth.test.ts (1 test) 6ms
 ✓ tests/npmTestRunsAllSuites.test.ts (1 test) 5ms
 ✓ tests/astroConfigLogLevel.test.ts (1 test) 4ms
 ✓ tests/prismSvelte.test.ts (1 test) 25ms
 ✓ scripts/tests/dbBenchmarkScript.test.ts (1 test) 6ms
 ✓ tests/progressionBalance.test.ts (1 test) 28ms
 ✓ tests/domException.test.ts (1 test) 5ms
 ✓ tests/indexeddb.server.test.ts (1 test) 273ms
 ✓ frontend/tests/sanity.test.ts (1 test) 3ms
All matched files use Prettier code style!
 ✓ scripts/tests/prettierFormatting.test.ts (1 test) 19186ms
   ✓ Prettier formatting > frontend sources are formatted  19184ms

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  tests/runRemoteCompletionistAwardIIIResolver.test.ts > Node version preflight > rejects unsupported versions and reports guidance
AssertionError: expected '[qa:remote-completionist-award-iii] U…' to contain '>=20 <22'

Expected: ">=20 <22"
Received: "[qa:remote-completionist-award-iii] Unsupported Node.js version 18.18.0. Required range: 20 || >=22. Run `nvm use` and reinstall dependencies with `pnpm install` before rerunning."

 ❯ tests/runRemoteCompletionistAwardIIIResolver.test.ts:67:21
     65|     expect(message).toContain('[qa:remote-completionist-award-iii]');
     66|     expect(message).toContain('18.18.0');
     67|     expect(message).toContain('>=20 <22');
       |                     ^
     68|     expect(message).toContain('nvm use');
     69|     expect(message).toContain('pnpm install');

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯


 Test Files  1 failed | 289 passed (290)
      Tests  1 failed | 1012 passed (1013)
   Start at  23:16:55
   Duration  167.61s (transform 10.21s, setup 11.70s, collect 45.97s, tests 109.87s, environment 190.12s, prepare 35.95s)


Error: AssertionError: expected '[qa:remote-completionist-award-iii] U…' to contain '>=20 <22'

Expected: ">=20 <22"
Received: "[qa:remote-completionist-award-iii] Unsupported Node.js version 18.18.0. Required range: 20 || >=22. Run `nvm use` and reinstall dependencies with `pnpm install` before rerunning."

 ❯ tests/runRemoteCompletionistAwardIIIResolver.test.ts:67:21


 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/src/pages/docs/md/contribute.md
+++ b/frontend/src/pages/docs/md/contribute.md
@@ -34,7 +34,7 @@ creation process.
 
 ## Code Contributions
 
-- [Submit a pull request](https://github.com/democratizedspace/dspace/pulls)
+- [Submit a pull request](https://github.com/democratizedspace/dspace/compare)
 - Fix bugs or implement new features
 - Improve documentation
 - Enhance the user interface

--- a/frontend/src/pages/docs/md/quest-guidelines.md
+++ b/frontend/src/pages/docs/md/quest-guidelines.md
@@ -317,7 +317,7 @@ Before submitting a quest, verify:
 2. Validate the quest JSON with `node scripts/validate-quest.js path/to/quest.json`.
 3. Update the matching Skills-category quest-tree doc (for example, `/docs/composting`) so it
    reflects any new quests or changed gates/rewards.
-4. Submit a [pull request](https://github.com/democratizedspace/dspace/pulls) with your quest JSON
+4. Submit a [pull request](https://github.com/democratizedspace/dspace/compare) with your quest JSON
    file.
 5. Respond to feedback during code review.
 

--- a/scripts/run-remote-completionist-award-iii.mjs
+++ b/scripts/run-remote-completionist-award-iii.mjs
@@ -17,24 +17,56 @@ const DEFAULT_BASE_URL = 'http://127.0.0.1:4173';
 const packageJson = JSON.parse(readFileSync(packageJsonPath, 'utf8'));
 const SUPPORTED_NODE_RANGE = packageJson?.engines?.node ?? '>=20 <22';
 
-function parseNodeMajorBounds(range) {
-  const lowerMatch = />=\s*(\d+)/.exec(String(range));
-  const upperMatch = /<\s*(\d+)/.exec(String(range));
+function parseNodeMajorRangeClause(clause) {
+  const comparators = String(clause)
+    .trim()
+    .split(/\s+/)
+    .filter(Boolean);
 
-  if (!lowerMatch || !upperMatch) {
+  if (comparators.length === 0) {
     return null;
   }
 
-  return {
-    minMajor: Number(lowerMatch[1]),
-    maxExclusiveMajor: Number(upperMatch[1]),
-  };
+  const checks = comparators.map((comparator) => {
+    const match = /^(>=|<=|>|<)?(\d+)$/.exec(comparator);
+    if (!match) {
+      return null;
+    }
+
+    const operator = match[1] ?? '=';
+    const major = Number(match[2]);
+
+    return (candidateMajor) => {
+      if (operator === '=') return candidateMajor === major;
+      if (operator === '>=') return candidateMajor >= major;
+      if (operator === '<=') return candidateMajor <= major;
+      if (operator === '>') return candidateMajor > major;
+      return candidateMajor < major;
+    };
+  });
+
+  if (checks.some((check) => check === null)) {
+    return null;
+  }
+
+  return (candidateMajor) => checks.every((check) => check(candidateMajor));
 }
 
-const SUPPORTED_NODE_MAJOR_BOUNDS = parseNodeMajorBounds(SUPPORTED_NODE_RANGE) ?? {
-  minMajor: 20,
-  maxExclusiveMajor: 22,
-};
+function parseNodeMajorRange(range) {
+  const clauses = String(range)
+    .split('||')
+    .map((clause) => parseNodeMajorRangeClause(clause))
+    .filter(Boolean);
+
+  if (clauses.length === 0) {
+    return null;
+  }
+
+  return (candidateMajor) => clauses.some((clause) => clause(candidateMajor));
+}
+
+const SUPPORTED_NODE_MAJOR_RANGE =
+  parseNodeMajorRange(SUPPORTED_NODE_RANGE) ?? ((major) => major >= 20 && major < 22);
 
 const require = createRequire(import.meta.url);
 
@@ -55,10 +87,7 @@ export function isSupportedNodeVersion(version = process.versions.node) {
   }
 
   const major = Number(majorMatch[1]);
-  return (
-    major >= SUPPORTED_NODE_MAJOR_BOUNDS.minMajor &&
-    major < SUPPORTED_NODE_MAJOR_BOUNDS.maxExclusiveMajor
-  );
+  return SUPPORTED_NODE_MAJOR_RANGE(major);
 }
 
 export function getUnsupportedNodeVersionMessage(version = process.versions.node) {

--- a/tests/runRemoteCompletionistAwardIIIResolver.test.ts
+++ b/tests/runRemoteCompletionistAwardIIIResolver.test.ts
@@ -64,7 +64,7 @@ describe('Node version preflight', () => {
     const message = getUnsupportedNodeVersionMessage('18.18.0');
     expect(message).toContain('[qa:remote-completionist-award-iii]');
     expect(message).toContain('18.18.0');
-    expect(message).toContain('>=20 <22');
+    expect(message).toContain('20 || >=22');
     expect(message).toContain('nvm use');
     expect(message).toContain('pnpm install');
   });
@@ -72,8 +72,8 @@ describe('Node version preflight', () => {
   it('accepts supported versions in the declared engine range', () => {
     expect(isSupportedNodeVersion('19.9.0')).toBe(false);
     expect(isSupportedNodeVersion('20.0.0')).toBe(true);
-    expect(isSupportedNodeVersion('21.9.0')).toBe(true);
-    expect(isSupportedNodeVersion('22.0.0')).toBe(false);
+    expect(isSupportedNodeVersion('21.9.0')).toBe(false);
+    expect(isSupportedNodeVersion('22.0.0')).toBe(true);
   });
 
   it('rejects malformed versions and accepts optional v-prefix versions', () => {

